slider/beatmap.py

```diff
@@ -450,6 +450,9 @@ def __init__(self,
                  repeat,
                  length,
                  ticks,
+                 num_beats,
```

Comment:
we need to keep the parameter list and the docstring in sync. I think the types should be:

```
num_beats : int
tick_rate : float
ms_per_beat : int
```

but I am not 100% sure.